packages/core-components/api-report.md

```diff
@@ -450,6 +450,8 @@ export type GaugeProps = {
   size?: 'normal' | 'small';
   description?: ReactNode;
   getColor?: GaugePropsGetColor;
+  relativeToMax?: boolean;
```

Comment:
Not sure about this one. If a max was given, when would you NOT want the gauge to be relative to the max? What are the behaviors when this is true vs false? My gut reaction is, this should always be true - right?

Reply:
Thank you for your feedback. I generally agree with you, and that is also what I would expect. However, there is one test which explicitly renders (value = 1, max = 10) as "1%": https://github.com/backstage/backstage/blob/master/packages/core-components/src/components/ProgressBars/Gauge.test.tsx#L40
The intention behind that test is not clear to me, so I opted not to change the behavior.

I would gladly tidy up more code, but without knowing why the code was written the way it is I didn't dare to. Please note that such a change also wouldn't be backwards compatible.

Reply:
Hi @freben, have you already had the opportunity to have a closer look?

In which way should I proceed? Keep by merge request as-is or make "relativeToMax" the default (if max exists), possibly breaking existing applications?